src/main/java/org/saltyrtc/client/cookie/Cookie.java

```diff
@@ -14,9 +14,6 @@
 
 /**
  * A SaltyRTC cookie.
- *
- * This data structure is immutable, but it can be cloned
- * using a copy constructor.
```

Comment:
Why did you remove this?

Reply:
This shouldn't be possible with an immutable class, I believe.

```
class Scratch {
    public static void main(String[] args) {
        Cookie cookie = new Cookie();
        byte[] cookieBytes = cookie.getBytes();
        System.out.println(Arrays.toString(cookieBytes));
        cookieBytes[0] = 54;
        System.out.println(Arrays.toString(cookie.getBytes()));
    }
}
```

Output:
```
[14, -97, 28, 37, -35, -33, 64, 59, 29, -36, 47, -118, 65, 68, 1, -47]
[54, -97, 28, 37, -35, -33, 64, 59, 29, -36, 47, -118, 65, 68, 1, -47]
```

Reply:
Yeah, that's a good point. Unfortunately Java does not give us write protection the way Rust does.

I guess it's ok if we simply remove the note. From the API it's already obvious that the values shouldn't be modified, I hope.